install_config/router/f5_router.adoc

```diff
@@ -26,6 +26,16 @@ like the
 xref:../../install_config/router/default_haproxy_router.adoc#install-config-router-default-haproxy[default
 HAProxy router].
 
+[IMPORTANT]
+====
+Support relationships between F5 and Red Hat provide a full scope of support for
+F5 integration. F5 provides support for the *F5 BIG-IP®* product, as it manages
+the code source for this integration. Both F5 and Red Hat jointly support the
```

Comment:
Let's keep this line as it is 'F5 provides support for the *F5 BIG-IP®* product.'